python/arcticdb/version_store/library.py

```diff
@@ -940,6 +940,56 @@ def read_metadata(self, symbol: str, as_of: Optional[AsOf] = None) -> VersionedI
         """
         return self._nvs.read_metadata(symbol, as_of)
 
+    def read_metadata_batch(
+        self, symbols: List[Union[str, ReadRequest]], query_builder: Optional[QueryBuilder] = None
+    ) -> List[VersionedItem]:
+        """
+        Reads the metadata of multiple symbols.
+
+        Parameters
+        ----------
+        symbols : List[Union[str, ReadRequest]
+            List of symbols to read.
+            Params columns, date_range and query_builder from ReadRequest are not used
```

Comment:
As mentioned before, this indicates that `ReadRequest` is not the right query type for this, we need a new one

Reply:
This has been discussed at length previously - we decided that this is a better solution then creating a seperate object for metadata methods.

Reply:
Why? I strongly disagree, these unused fields are one of the most typical flags of incorrect API design.

Reply:
As we discussed, another class field is added, common to both read_metadata_batch and read_descriptor_batch